docs/knowledgebase/integrate/subkey.md

```diff
@@ -298,39 +287,38 @@ Secret Key URI `image stomach entry drink rice hen abstract moment nature broken
 
 ```bash
 # SS58-address plus derivation path
-$ subkey inspect "5CZtJLXtVzrBJq1fMWfywDa6XuRwXekGdShPR4b8i9GWSbzB/joe/polkadot/0"
+$ subkey inspect-key --uri "5CZtJLXtVzrBJq1fMWfywDa6XuRwXekGdShPR4b8i9GWSbzB/joe/polkadot/0"
 Public Key URI `5CZtJLXtVzrBJq1fMWfywDa6XuRwXekGdShPR4b8i9GWSbzB/joe/polkadot/0` is account:
-  Network ID/version: substrate
+  Network ID/version: polkadot
   Public key (hex):   0xc69291081743ba7e8f587f1e848d1ffc7b634ce21793a231b7ac75d430b59068
   Account ID:         0xc69291081743ba7e8f587f1e848d1ffc7b634ce21793a231b7ac75d430b59068
-  SS58 Address:       5GZ4srnepXvdsuNVoxCGyVZd8ScDm4gkGLTKuaGARy9akjTa
+  SS58 Address:       15VN2C3igKC7KSP1mbFH7ePmz4bsTNEtLqBp4sFWz4B6wJsa
 ```
 
-Notice that the "SS58-address plus derivation path" produces the same address as the "mnemonic
-phrase plus derivation path plus password." As such, you can reveal your parent address and
-derivation paths without revealing your mnemonic phrase or password, retaining control of all
-derived addresses.
-
```

Comment:
@shawntabrizi, @joepetrowski - also not working :thinking:

Reply:
Same issue with default network change.